datajunction-clients/python/tests/test_integration.py

```diff
@@ -737,3 +505,6 @@ def test_integration():  # pylint: disable=too-many-statements,too-many-locals,l
 
     # Check that publishing transform 3 works
     transform_3.publish()
+
+    # Delete the test namespace
+    dj.delete_namespace(namespace=namespace, cascade=True)
```

Comment:
Not related to this PR, but I wonder if we need a hard-delete for namespaces as well? It's not ideal that running the integration test locally (or even worse, someone could run this against an actual deployed environment too) would create a bunch of deactivated namespaces at the end. Ideally these would be fully deleted

Reply:
Oh yes, that would be useful to have and this integration test would be a good use case. 
Made an issue: https://github.com/DataJunction/dj/issues/744